Drivers/Src/LTC6811.c

```diff
@@ -92,13 +115,17 @@ void LTC6811_Init(cell_asic *battMod){
     mutexExists = true;
   }
 
```

Comment:
In here, I think you would have to have an ifdef which sets the pend and post to empty functions for baremetal. EX.
/***BAREMETAL VERSION OF CODE**/
LTC6811_Pend(){ return;}
So you would have two pend and post functions and depending on which version we are compiling. That is the version that would be placed in the struct.